pythran/dist.py

```diff
@@ -7,44 +7,40 @@
 
 import os.path
 import os
+from warnings import warn
```

Comment:
There's already a logging mechanism in pythran, with a warning level, please use it :-)